R/bounds.r

```diff
@@ -155,56 +155,91 @@ rescale_none <- function(x, ...) {
   x
 }
 
-#' Censor any values outside of range
+#' @name oob
+#' @aliases oob_censor oob_censor_any censor oob_squish oob_squish_any squish
+#'   oob_squish_infinite squish_infinite oob_discard discard oob_keep
+#' @title Out of bounds handling
+#' @description This set of functions modify data values outside a given range.
+#'
+#' @param x A numeric vector of values to modify.
+#' @param range A numeric vector of length two giving the minimum and maximum of
+#'   the desired output range respectively.
+#' @param only.finite A logical of length one. When `TRUE`, only finite values
+#'   are altered. When `FALSE`, also infinite values are altered.
+#'
+#' @return Most `oob_()` functions return a vector of numerical values parallel
+#'   to the `x` argument, wherein out of bounds values have been modified. Only
+#'   `oob_discard()` returns a vector of less than or of equal length to the `x`
+#'   argument.
+#'
+#' @details The `oob_*()` functions are designed to be passed as `oob` argument
```

Comment:
I'd suggest combining this with `@description` (so it's "above the fold" in the docs) and use a bulleted list to make it easier to scan the functions